blog/schema.sql

```diff
@@ -7,6 +8,11 @@ CREATE TABLE user (
     password TEXT NOT NULL
 );
 
+CREATE TABLE user_email (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    email TEXT UNIQUE NOT NULL
+);
+
```

Comment:
Nitpicks for the `user_email` table:

1. `email` is a candidate key on its own (it's unique), so we should get rid of the `id` field if possible.
2. could you give it a different name (`newsletter_subscriber` or something?) To me the name `user_email` suggests one of two things: either a table that is one-to-many with `user` (i.e. if a user could have multiple emails, `user_email` would hold each of their emails) or a joining table between two many-to-many tables `user` and `email`